osv/ecosystems/haskell.py

```diff
@@ -69,7 +69,7 @@ class GHC(Ecosystem):
   """Glasgow Haskell Compiler (GHC) ecosystem."""
 
   _API_PACKAGE_URL = ('https://gitlab.haskell.org'
-                      '/api/v4/projects/3561/repository/tags?per_page=-1')
+                      '/api/v4/projects/3561/repository/tags?per_page=100')
```

Comment:
maybe add a comment saying that 100 is `max` for `per_page`?

Reply:
comment added